stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java

```diff
@@ -218,32 +218,54 @@ public TaskResult.Aggregator aggregator() {
   }
 
   private static final class Aggregator implements TaskResult.Aggregator<ClientIOTaskResult> {
+    @SuppressWarnings("checkstyle:OperatorWrap")
     @Override
     public ClientIOSummary aggregate(Iterable<ClientIOTaskResult> results) throws Exception {
       long recordStartMs = 0;
       long endMs = 0;
-      long ioBytes = 0;
       ClientIOParameters clientIOParameters = null;
       BaseParameters baseParameters = null;
       Map<String, ClientIOTaskResult> nodes = new HashMap<>();
-
       for (ClientIOTaskResult taskResult: results) {
-        recordStartMs = taskResult.getRecordStartMs();
-        endMs = Math.max(endMs, taskResult.getEndMs());
-        for (ThreadCountResult result: taskResult.getThreadCountResults().values()) {
```

Comment:
I was summing up throughput of all tested thread numbers, which doesn't make sense